src/OrderPage.js

```diff
@@ -1,22 +1,88 @@
-import React from "react"
-import StatusPanel from "./StatusPanel";
-import {Button} from "@material-ui/core";
+import React from "react";
+import {Button, Paper} from "@material-ui/core";
 import OrderFilter from "./OrderFilter";
 import OrderTable from "./OrderTable";
+import Grid from "@material-ui/core/Grid";
+import {ORDERS} from "./ordersData";
+
+const styles = {
+  paper: {
+    margin: '10px',
+    padding: '10px'
+  },
+};
 
 class OrderPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      contractor: [],
+      orderStatus: [],
+      selectedFromDate: null,
+      selectedTillDate: null,
+    };
+    this.filteredOrdersArray = null;
+  }
+
+  handleContractorSelect = event => {
+    this.setState({contractor: event.target.value});
+  };
+  handleOrderStatusSelect = event => {
+    this.setState({orderStatus: event.target.value});
+  };
+  handleFromDateChange = date => {
+    this.setState({selectedFromDate: date});
+  };
+  handleTillDateChange = date => {
```

Comment:
this.state = {currentFilters: []}
handleFilter = (param) =>{
const newCurrentFilters = [
...this.state.currentFilters,
param
]
}